exe/kubernetes-deploy

```diff
@@ -9,7 +9,23 @@ skip_wait = false
 template_dir = nil
 allow_protected_ns = false
 prune = true
+bindings = {}
+
+def blank?(v)
```

Comment:
This defined global `Kernel.blank?` which may somehow conflict with ActiveSupport. Can you move it into a helper module?